test/integration/cni/host_networking_test.go

```diff
@@ -36,13 +36,27 @@ const (
 	NEW_VETH_PREFIX            = "veth"
 	DEFAULT_MTU_VAL            = "9001"
 	DEFAULT_VETH_PREFIX        = "eni"
+	WARM_IP_TARGET             = "WARM_IP_TARGET"
```

Comment:
This can be removed, should be good to use "WARM_IP_TARGET" string directly in argument below.

Reply:
Since the other variables used global const strings, I figured it was best to keep it consistent. Then only one line can be changed in the future

Reply:
I think we use ENV names like `WARM_IP_TARGET` as strings in other places too. Not required to use a const for it here.